projects/graph/src/draw.py

```diff
@@ -2,13 +2,172 @@
 General drawing methods for graphs using Bokeh.
```

Comment:
Consider adding *shebangs* to the top of each Python file, just to get in the habit of it for when you get a job. They may seem like they aren't that important but they are a central part of the Python methodology.